### DISCUSSION TITLE
Changes in code on local machine are not getting reflected on the localhost

### DISCUSSION THREAD

**Discussion by Sudarshan-21** [processing/p5.js-web-editor#2791] [General]
![Screenshot (75)](https://github.com/processing/p5.js-web-editor/assets/137636943/823ba5fc-1b93-4589-bbbd-ffb09a2c42b6)

### What is your operating system?

windows 11

### Web browser and version

chrome://120.0.6099.72

### Actual Behavior

when p5.js code is run on local machine and changes are made, they are not getting reflected on localhost.

### Expected Behavior

if changes are made in the code the respective changes should reflect on the localhost

### Steps:
1. make changes in the p5.js code on local machine
2. reload the localhost
3. open console

**Comment by welcome** [comment]
Welcome! 👋 Thanks for opening your first issue here! And to ensure the community is able to respond to your issue, be sure to follow the issue template if you haven't already.

**Comment by skushagra9** [comment]
What is the error you are facing ?

**Comment by Sudarshan-21** [comment]
i am not able to see changes in the p5.js local machine setup when i change code on the local machine, i have attached the screenshot for reference

**Comment by HrxSrv** [comment]
please specify your docker version.

**Comment by Sudarshan-21** [comment]
docker version: 24.0.7

**Comment by Sudarshan-21** [comment]
@skushagra9 can you help?

**Comment by skushagra9** [comment]
where have you made the changes ?

**Comment by rahulrangers** [comment]
yeah i am also facing same issue i have changed i name Cookies into Cookies222 and i am not able to see the change.

**Comment by skushagra9** [comment]
when you restart server , are you able to see then ?

**Comment by rahulrangers** [comment]
yes i can see them when i restart the container and , when i enter the docker container(by the command docker exec -it <id>) i can see the files inside are being changed inside the container but don't know why are not being reflected into the localhost.

**Comment by Saikat123Dev** [comment]
You can follow the manual installation 
Follow this docs https://github.com/processing/p5.js-web-editor/blob/develop/contributor_docs/installation.md.
Don't use docker in case of windows it will not reflect the changes in localhost .

**Comment by Sudarshan-21** [comment]
@rahulrangers you should consider switching to other operating system. I did the same, it runs fine on linux

**Comment by rahulrangers** [comment]
i tried manual installation and it is working thanks for your help :)
